go.mod

```diff
@@ -3,35 +3,36 @@ module github.com/cosmos/gaia/v6
 go 1.17
 
 require (
-	github.com/cosmos/cosmos-sdk v0.44.3
-	github.com/cosmos/ibc-go/v2 v2.0.0
+	github.com/cosmos/cosmos-sdk v0.44.5
+	github.com/cosmos/ibc-go/v2 v2.0.1
 	github.com/gorilla/mux v1.8.0
 	github.com/gravity-devs/liquidity v1.4.2
 	github.com/rakyll/statik v0.1.7
 	github.com/spf13/cast v1.4.1
 	github.com/spf13/cobra v1.2.1
 	github.com/strangelove-ventures/packet-forward-middleware v1.0.1
 	github.com/stretchr/testify v1.7.0
-	github.com/tendermint/tendermint v0.34.14
-	github.com/tendermint/tm-db v0.6.4
+	github.com/tendermint/tendermint v0.34.15
```

Comment:
tendermint 0.34.15 need key migration. We need to integrate [pre-upgrade](https://github.com/cosmos/cosmos-sdk/issues/9973) to the app ?

Reply:
Hey @yaruwangway! I think that the change you're referring to is the data base key change that occurred as part of [v0.35](https://github.com/tendermint/tendermint/blob/a97b081df100cb43398fabb925ffae3f35b9816a/UPGRADING.md#database-key-format-changes). Which means that that change does not play a role in updating to Tendermint v0.34.15.

The change log for v0.34.15 was mostly [bug fixes](https://github.com/tendermint/tendermint/blob/41c176ccc6a75d25631d0f891efb2e19a33329dc/CHANGELOG.md#L11)

Reply:
ha, right! thanks!